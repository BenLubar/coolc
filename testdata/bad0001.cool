class Main() extends IO("foo") {
}
