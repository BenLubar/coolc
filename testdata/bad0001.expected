testdata/bad0001.cool:1:22: wrong number of method arguments
basic.cool:26:7: (method is declared here)
